lib/request.coffee

```diff
@@ -322,4 +306,50 @@ module.exports = getRequest = ({
 	# (or resolve to) a new response, or throw/reject.
 	###
 
+	###*
+	# @summary Refresh token on user request
+	# @function
+	# @public
+	#
+	# @description
+	# This function automatically refresh token on user request.
+	# The token is refreshed even if it is not expired yet.
+	# Error is thrown if user is not authorized.
```

Comment:
I don't think this is necessary to document, as it will only occur if the user tries to run this method without correctly initializing the module